main.py

```
@@ -55,15 +86,23 @@
-# Initialisation DB (création des tables si nécessaire)
+# Initialisation DB et JWT
+jwt = JWTManager(app)
+
 db.init_app(app)
 with app.app_context():
     try:
         db.create_all()
-    except Exception as e:
-        # En environnement serverless, échouer silencieusement si non nécessaire
+    except Exception:
         pass
```

Ruff (0.13.1)

`try`-`except`-`pass` detected, consider logging the exception

(S110)